cogs/links.py

```diff
@@ -108,6 +111,38 @@ def __init__(self, bot):
             await ctx.send(f'{ctx.message.author.mention}', embed=embed)
             return
 
+    @commands.command(name='monster',
+                      description='Lookup osrs monsters.',
+                      aliases=['qs'],
+                      case_insensitive=True)
+    async def quest_command(self, ctx, *monster):
+        """ Looks up a monster """
+        monster = " ".join(monster)
+        loaded_monster = load_monster_from_api(monster.lower())
+        if loaded_monster is not False:
+            stats = f"`Combat level:` {loaded_monster.combat_level} \n"
+            stats += f"`Hitpoints:` {loaded_monster.hitpoints} \n"
+            stats += f"`Max hit:` {loaded_monster.max_hit} \n"
+            stats += f"`Aggressive:` {'Yes' if loaded_monster.aggressive else 'No'} \n"
+            stats += f"`Attack speed:` {loaded_monster.attack_speed} ticks \n"
+            stats += f"`Weakness:` {'None' if len(loaded_monster.weakness) == 0 else ', '.join([x for x in loaded_monster.weakness])} \n "
+            stats += f"`Attack style:` {', '.join([x for x in loaded_monster.attack_type])} \n"
+            loaded_monster_drops = parse_monster_drops(loaded_monster.drops)
+            drops = ""
+            index = 0
+            for drop in range(5):
+                index += 1
+                drops += f"`{loaded_monster_drops[index]['name']}:` {loaded_monster_drops[index]['value']}gp \n"
```

Comment:
Just realised that this isnt very clean code.
It needs to be drapped in to and if else block and the code can be shortened to 2 lines with list comprehension.